packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -117,8 +117,13 @@ export class CeramicCliUtils {
       : DEFAULT_DAEMON_CONFIG_FILENAME
     const config = await this._loadDaemonConfig(configFilepath)
 
+    // Environment variables override values from config file
+    if (envVarExists(process.env.CERAMIC_INDEXING_DB_URI)) config.indexing.db = process.env.CERAMIC_INDEXING_DB_URI
```

Comment:
This could be simplified to just
```suggestion
    if (process.env.CERAMIC_INDEXING_DB_URI) config.indexing.db = process.env.CERAMIC_INDEXING_DB_URI
```

An empty string "", and `undefined` are both coerced to a boolean `false`.